src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -8,6 +8,16 @@
 
 namespace System
 {
+   public partial class AppDomain
```

Comment:
> everything in System.Runtime.Extensions gets folded into System.Runtime 

Would it make sense to keep System.Runtime.Extensions for things like AppDomain? Fold just what make sense - not everything.